src/localisation/currency.h

```diff
@@ -51,6 +51,7 @@ typedef enum {
 
 // Currency format specification - inspired by OpenTTD
 typedef struct {
+	char isoCode[3];
```

Comment:
You can't set a char array to a string literal. You would have to either define it as a const char * or define it like { 'G', 'B', 'P', '\0' }.

Unless there is some other syntax that allows string literal for char array.
I know that you can do ```char stuff[] = "the string"``` which works, but you can't do that in the table above.